README.md

```diff
@@ -39,36 +55,70 @@ Several choices are offered to you:
 
 ### Get started with electron?
 
-#### Importing
+#### Install dependencies
+
+First of all, you will need to clone this current repository, after that use a package manager to install dependencies. If you are using `yarn`, use the following commands:
 
-First of all, you will need to clone this current repository, after that use a package manager to install electron forge. If you are using `yarn`, use the following commands:
 ```bash
-$ cd /path/to/Zeal-WebEmulator
-$ yarn add --dev @electron-forge/cli
-$ yarn electron-forge import
+cd /path/to/Zeal-WebEmulator
+yarn install
 ```
 
 If you are using `npm`, the commands are as follows:
+
 ```bash
-$ cd /path/to/Zeal-WebEmulator
-$ npm install --save-dev @electron-forge/cli
-$ npm exec --package=@electron-forge/cli -c "electron-forge import"
+cd /path/to/Zeal-WebEmulator
+npm install
 ```
 
-If you encounter any issues, you can find more information on [the official electron forge website.](https://www.electronforge.io/import-existing-project)
+If you encounter any issues, you can find more information on [the official electron forge website](https://www.electronforge.io/import-existing-project).
 
 #### Running
 
-Once the project has been imported, it can be executed thanks to
-```
-$ yarn start
+Once the dependencies are ready, it can be executed thanks to
+
+```bash
+yarn start
 ```
+
 or
+
+```bash
+npm start
 ```
-$ npm start
-```
+
 commands.
 
+#### Publishing
```

Comment:
What is the purpose of pusblishing?

Reply:
Sorry, this title is a little confusing, the script will build a binary. I should call it "compiling" or someelse.

Reply:
Fixed.